lib/hq/graphql/ext/schema_extensions.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module HQ
+  module GraphQL
+    module Ext
+      module SchemaExtensions
+        def self.prepended(klass)
+          klass.alias_method :add_type_without_lazyload, :add_type
+          klass.alias_method :add_type, :add_type_with_lazyload
+        end
+
+        def execute(*args, **options)
+          lazy_load!
```

Comment:
GraphQL::Execution::Interpreter doesn't use `to_graphql` to cache types which pushes back type generation until the very first query. This means that the first query is going to be slow if you have a very large schema. Workaround for this is to forcefully load types after your web server loads.

Unicorn example:
```ruby
after_fork do
  ::AgencieshqGraphqlSchema.lazy_load!
end
```